controllers/cluster_controller_test.go

```diff
@@ -260,9 +260,22 @@ var _ = Describe("Cluster Reconciler", func() {
 			return len(cluster.Finalizers) > 0
 		}, timeout).Should(BeTrue())
 
+		// Create a node so we can speed up reconciliation. Otherwise, the machine reconciler will requeue the machine
+		// after 10 seconds, potentially slowing down this test.
```

Comment:
Not sure if this is the root cause of our flake. Initially I thought it was, but now I'm less certain.